GameData/RealismOverhaul/REWORK/RO_CZ3B.cfg

```diff
@@ -440,9 +440,6 @@
 	%title = YF-75 (2x) [3 m]
 	%manufacturer = CALT
 	%description = Chinese gas generator cryogenic upper stage engine, used on the Long March series of rockets in a double configuration. Can restart up to two times.
-	{
```

Comment:
I think this is missing a @MODEL, rather than the node needing to be deleted.

Reply:
OK, I'll try it that way instead. Is this what you had in mind?

```
	@MODEL
	{
		%scale = 1, 1, 1
	}
```